S1_NRB/nrb.py

```diff
@@ -517,9 +524,6 @@ def create_rgb_vrt(outname, infiles, overviews, overview_resampling):
         color = etree.Element('ColorInterp')
         color.text = col
         band.insert(0, color)
-    for i, band in enumerate(bands):
-        if i in [0, 1]:
-            band.remove(band.find('NoDataValue'))
```

Comment:
Hi @maawoo. Thanks for having a look at this. I wanted to ask you why you put it in there before I merge the PR. The visualization without those lines looked really good in my case.

Reply:
Hi John! 
Sorry that it took me so long to get back to this. 
Your changes to `create_rgb_vrt` result in a broken VRT on my end (QGIS 3.24.1-Tisler with GDAL 3.3.2):

<img width="1000" alt="Screenshot 2022-06-20 at 09 59 54" src="https://user-images.githubusercontent.com/56583917/174553678-f8a9629c-f19f-4755-b048-e35f6b645d75.png">

While the "old" VRT based on my code is visualizing correctly, I realize that my approach was also flawed.
I modified the "new" VRT and it would be great if you could check that it also works on your end. The following archive includes the necessary GeoTIFF files, the "new" VRT and a modification of it:
https://upload.uni-jena.de/data/62b033338612b3.83100075/S1B_IW_NRB__1SDV_20210514T051703_026896_033696_33UUS_4C86.zip

These are the changes I made in the modified VRT:
- Remove `NODATA` from all bands
- Change `NoDataValue` to -9999 for bands 1, 2 and the nested VRT in band 3

From the [GDAL docs](https://gdal.org/drivers/raster/vrt.html#vrtrasterband):
>NoDataValue: If the input datasets to be composed have a nodata value for this raster band, set this element’s value to that nodata value for it to be reflected in the VRT. This must not be confused with the NODATA element of a VRTComplexSource element.

As far as I'm concerned, `NODATA` can be ignored in our case (see also [this response](https://github.com/OSGeo/gdal/issues/4746#issuecomment-956441213)).